src/widget/weffectchainpresetselector.cpp

```diff
@@ -37,7 +37,7 @@ void WEffectChainPresetSelector::setup(const QDomNode& node, const SkinContext&
     }
 
     auto chainPresetListUpdateSignal = &EffectChainPresetManager::effectChainPresetListUpdated;
-    auto pQuickEffectChain = qobject_cast<QuickEffectChain*>(m_pChain.data());
+    auto* pQuickEffectChain = qobject_cast<QuickEffectChain*>(m_pChain.data());
     if (pQuickEffectChain) {
```

Comment:
Is it clean to use `if (pQuickEffectChain) {` or should it better be `if (pQuickEffectChain!=nullptr) {` ?

Reply:
Both works equal good. We have no preference for one or the other. I personal prefer the implicit bool cast.

Reply:
Ok